sale_partner_shipping_invoice_domain/readme/USAGE.rst

```diff
@@ -0,0 +1,3 @@
+Go to settings -> enable multiple contact addresses.
+
+Go to a sale order form view. Choose a customer then use the fields for delivery and invoicing addresses.
```

Comment:
Could you enhance a little bit, to say that it will restrict invoice and delivery address depending on commercial partner (and finally the selected partner - first field).

Reply:
Thanks for your feedback, it is updated